### PR TITLE
fix: typo in sip-192

### PR DESCRIPTION
**Pull request by drptbl**
Fixed typo in SIP-192.

### CODE REVIEW

**Review by dbeal-eth — APPROVED** [review]
:+1: 

also can you fix the submitted date? that was jsut copied and pasted from another one and I think its wrong also.

**Comment by drptbl** [comment]
@dbeal-eth done, thanks :+1:

File: content/sips/sip-192.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ author: Daniel Beal (@dbeal-eth)
 implementor: Daniel Beal (@dbeal-eth)
 release: 12-2-2021
 discussions-to: 'https://research.synthetix.io'
-created: 2021-11-18T00:00:00.000Z
+created: 2021-12-02T00:00:00.000Z
```
Comment (andytcf) [inline content/sips/sip-192.md:10]: probs put the a new field here @drptbl 

created: 2021-11-18T00:00:00.000Z
updated: 2021-12-02T00:00:00.000Z

**Review by andytcf — DISMISSED** [review]

File: content/sips/sip-192.md (hunk 1 above)
Comment (drptbl) [reply content/sips/sip-192.md:10]: @andytcf are you sure?
It wasn't really created on 2021-11-18 afaik.

**Review by justinjmoses — APPROVED** [review]

**Pull request merged**